automatic/musescore/musescore.nuspec

```diff
@@ -7,7 +7,7 @@
     <title>MuseScore</title>
     <owners>chocolatey-community</owners>
     <authors>MuseScore developers</authors>
-    <licenseUrl>https://github.com/musescore/MuseScore/blob/master/LICENSE.GPL</licenseUrl>
+    <licenseUrl>https://github.com/musescore/MuseScore/blob/master/LICENSE.txt</licenseUrl>
```

Comment:
Looks like the license at this location is not equal to the embedded license we have (probably just the dates being wrong).

Could you update the embedded license in the legal folder as well?